src/main/java/org/eln2/mc/common/damagehandlers/IDamageHandler.kt

```diff
@@ -0,0 +1,5 @@
+package org.eln2.mc.common.damagehandlers
+
+class IDamageHandler {
+
+}
```

Comment:
Is this an interface? No brackets needed.

Reply:
Removed the class.